kbfsgit/fs.go

```diff
@@ -188,8 +199,27 @@ func (fs *FS) Open(filename string) (billy.File, error) {
 }
 
 // Stat implements the billy.Filesystem interface for FS.
-func (fs *FS) Stat(filename string) (os.FileInfo, error) {
-	return nil, nil
+func (fs *FS) Stat(filename string) (fi os.FileInfo, err error) {
+	fs.log.CDebugf(fs.ctx, "Stat %s", filename)
+	defer func() {
+		fs.deferLog.CDebugf(fs.ctx, "Stat done: %+v", err)
+	}()
+
+	n, base, err := fs.lookupParent(filename)
+	if err != nil {
+		return nil, err
+	}
+
+	_, ei, err := fs.config.KBFSOps().Lookup(fs.ctx, n, base)
+	if err != nil {
+		return nil, err
+	}
+
+	return &FileInfo{
+		fs:       fs,
+		ei:       ei,
+		fullpath: filename,
```

Comment:
Did you intend to have `fullpath` always include full path or does it not matter? It seems it's only used by logging though.